composer/models/huggingface.py

```diff
@@ -718,6 +836,59 @@ def write_huggingface_pretrained_from_composer_checkpoint(
     config = get_hf_config_from_composer_state_dict(composer_state_dict)
     config.save_pretrained(output_folder)
 
+    peft_config = get_peft_config_from_composer_state_dict(composer_state_dict)
+    if peft_config is not None:
+        peft_config.save_pretrained(str(output_folder))
+
     weights_state_dict = composer_state_dict['state']['model']
     torch.nn.modules.utils.consume_prefix_in_state_dict_if_present(weights_state_dict, prefix='model.')
-    torch.save(weights_state_dict, Path(output_folder) / 'pytorch_model.bin')
+
+    # NOTE: This only works for default adapter name
```

Comment:
Technically peft models can have multiple named adapters (the default name is `default`). The way I wrote this function just hardcoded `default` because the way it figured out which adapter to use comes from the `PeftModel.active_adapter`, but I don't have a `PeftModel` everywhere I need this function. Since we're not trying to support all PEFT features at this time, its just hardcoded to default. But let me generalize the implementation a little bit to make it easier to add support in the future.

Reply:
refactored to take in the argument name (even though this code will only set it to default), and updated the comment a bit